docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -19,6 +19,116 @@ If applicable, exact error messages linked to this issue so users searching for
 Link to fix
 ////
 
+[[broken-apm-anomaly-rule]]
+*Upgrading to v8.13 breaks APM anomaly rules* +
+_Elastic Stack versions: 8.13.0, 8.13.1, 8.13.2_ +
+_Fixed in Elastic Stack version 8.14.0_
+
+// The conditions in which this issue occurs
+This issue occurs when upgrading the Elastic Stack to version 8.13.0 or 8.13.1.
```

Comment:
```suggestion
This issue occurs when upgrading the Elastic Stack to version 8.13.0, 8.13.1, or 8.13.2.
```
Should 8.13.2 be added here since it's in the list of impacted versions?